### PR TITLE
Add performance Tests and Health Checks

### PR DESCRIPTION
**Pull request by 15Dkatz**
:potable_water:

Add some documentation related to the Finatra [HTTP Admin interface](https://twitter.github.io/twitter-server/Admin.html#admin-interface). With this interface, we can run convenient and comprehensive checks on diagnostics, profiling, and much more.

This code change also enables performance testing with `jmeter`, an added tool inspired by the [pravda project](https://github.com/zendesk/pravda).

/cc @zendesk/sylvius

### Reference:
JIRA: https://zendesk.atlassian.net/browse/TR-51

### How could this be manually tested?
* Make sure the server is running: `sbt run`
* Visit `http://localhost:9990/admin`
* Install jmeter: `brew install jmeter`
* Run jmeter: `jmeter`
* Load the `DecoSummaryReport.jmx` file and run the Jmeter tests (See updated README for more details)

### Risks
* None

### CODE REVIEW

File: DecoSummaryReport.jmx [hunk 1]
```diff
@@ -0,0 +1,141 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (smedberg) [inline DecoSummaryReport.jmx]: Just a thought- could this file be moved to a directory called `jmeter` or something like that?  Or could the name of it be changed to reflect that it's for use with jmeter?  When I first glanced at it, I assumed it was the output of a test run, and should not be checked in (that was wrong, as I now understand it.)

Comment (15Dkatz) [inline DecoSummaryReport.jmx]: Agreed. I think placing it in `jmeter/plans` will represent that it's a specific file used to run jmeter.

**Comment by 15Dkatz** [comment]
💇‍♂️ @smedberg

File: README.md [hunk 2]
```diff
@@ -12,13 +12,34 @@ Deco, short for "decorator", is an attribute service for Zendesk objects. For ex
...
-This project uses [Finatra](https://twitter.github.io/finatra/) as its web framework.
+We use jmeter to run performance tests. First, make sure the server is running: `sbt run`.
+
+Then install and start jmeter:
+```
+brew install jmeter
+jmeter
+```
+
+In the Jmeter GUI, load `jmeter/plans/DecoSummaryReport.jmx` and click `run`. Check out the results in the left pane by clicking `View Results in Table`.
```
Comment (smedberg) [inline README.md]: nit: In my version of jmeter, the button is labeled "start", rather than "run", but it's not too confusing :)

**Comment by 15Dkatz** [comment]
@smedberg 💇‍♂️ 

I updated it to `Start` :) The dropdown was labeled `Run` but I think `Start` is more clear since that's the actual button.

**Pull request merged**